src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,47 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ description, imdbUrl, imgUrl, title }) => (
+  <>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="./images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{title}</p>
+        </div>
+      </div>
+
+      <div className="content">
+        {description}
+        <br />
+        <a href={imdbUrl}>IMDB</a>
+      </div>
+    </div>
+  </>
 );
+
+MovieCard.defaultProps = {
+  title: 'no title',
+};
+
+MovieCard.propTypes = {
+  description: PropTypes.string.isRequired,
+  imdbUrl: PropTypes.string.isRequired,
+  imgUrl: PropTypes.string.isRequired,
+  title: PropTypes.string,
+};
```

Comment:
By the terms of the task, all the fields except the description are required.  
Swap description and title, and then write default value for description)